### PR TITLE
Docs: Rewrite alerting docs

### PR DESCRIPTION
**Pull request by oddlittlebird**
Rewrote the alerting docs to be valid for Grafana 7.0. Also added a few missing tasks. We'll need to go back and overhaul notifications.md at some point, but that is out of scope for 7.0.

### CODE REVIEW

File: docs/sources/menu.yaml [hunk 1]
```diff
@@ -204,12 +204,16 @@
...
+    - link: /alerting/alerts-overview/
+      name: Overview
     - link: /alerting/notifications/
-      name: Notifications
+      name: Alert notifications
+    - link: /alerting/create-alerts/
+      name: Create alerts
+    - link: /alerting/view-alerts/
+      name: View alerts
+    - link: /alerting/troubleshoot-alerts/
```
Comment (marefr) [inline docs/sources/menu.yaml:215]: Pause an alert page should be included in alerting menu, right?

**Review by marefr — CHANGES_REQUESTED** [review]
Looks great :+1: Some minor comments/suggestions before we can merge this.

File: docs/sources/alerting/create-alerts.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+In the Alert tab of the graph panel you can configure how often the alert rule should be evaluated and the conditions that need to be met for the alert to change state and trigger its [notifications]({{< relref "notifications.md" >}}).
+
+Currently only the graph panel supports alert rules.
+
+## Add or edit an alert rule
+
+1. Navigate to the panel you want to add or edit an alert rule for, click the title, and then click **Edit**.
+1. On the Alert tab, click **Create Alert**. If an alert already exists for this panel, then you can just edit the fields on the Alert tab.
+1. Fill out the fields. Descriptions are listed below in [Alert rule fields](#alert-rule-fields).
+1. When you have finished writing your rule, click **Test rule** to make sure the rule returns the results you expect.
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/alerting/create-alerts.md:27]: I did not realize that! Added a save step before the test.

File: docs/sources/alerting/alerts-overview.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+title = "Alerts overview"
+type = "docs"
+[menu.docs]
+identifier = "alerting"
+parent = "Alerting"
+aliases = ["/docs/grafana/latest/alerting/rules/", "/docs/grafana/latest/alerting/metrics/"]
+weight = 100
++++
+
+# Alerts overview
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/alerting/alerts-overview.md:11]: Literally the first word of this topic is Alerting. Will change it to Grafana Alerting.

[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/alerting/alerts-overview.md:11]: Also, this particular part refers to the Alert tab and the Create Alert button.

File: docs/sources/alerting/create-alerts.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+## Alert rule fields
+
+This section describes the fields you fill out to create an alert.
+
+### Rule
+
+- **Name -** Enter a descriptive name. The name will be displayed in the Alert Rules list.
+- **Evaluate every** and **For -**  Specify the name of the alert rule and how often the scheduler should evaluate the alert rule. 
+
+You can set a minimum interval in the `alerting.min_interval_seconds` config field, to set a minimum time between evaluations. Refer to [Configuration]({{< relref "../installation/configuration.md" >}}#min-interval-seconds) for more information.
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/alerting/create-alerts.md:42]: Yes, it is exactly that except that I am using the exact field name and I edited the content slightly.

File: docs/sources/alerting/create-alerts.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+> **Caution:** Do not use `For` with the `If no data or all values are null` setting set to `No Data`. The triggering of `No Data` will trigger instantly and not take `For` into consideration. This may also result in that an OK notification not being sent if alert transitions from `No Data -> Pending -> OK`.
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/alerting/create-alerts.md:44]: Literally did not understand that from the previous writeup. Good add.

File: docs/sources/alerting/pause-an-alert.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+type = "docs"
+[menu.docs]
+name = "Pause alerts"
+parent = "alerting"
+weight = 400
++++
+
+# Pause an alert
+
+Once you’ve acknowledged an alert, consider pausing it. This can be useful to avoid sending subsequent alerts while you work on a fix.
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/alerting/pause-an-alert.md:14]: I literally took that line from the previous writeup. Thank you so much for looking this over!

File: docs/sources/menu.yaml (hunk 1 above)
Comment (oddlittlebird) [reply docs/sources/menu.yaml:215]: You can tell I was getting tired, lol. Good catch.

**Review by bergquist — APPROVED** [review]

**Review by marefr — APPROVED** [review]
LGTM, menu.yaml still needs a quick update (see comments)

File: docs/sources/menu.yaml [hunk 7]
```diff
@@ -204,12 +204,16 @@
...
+      name: Troubleshoot alerts
```
[the remark this answers is not in the document]
Comment (oddlittlebird) [reply docs/sources/menu.yaml:218]: Hmm. It is showing on mine and looks fine in my local build. Weird.

**Pull request merged**